proto/api/v1/common.proto

```diff
@@ -8,3 +8,11 @@ package v1;
 message ResourceByID {
     string id = 1;
 }
+
+// EXPERIMENTAL.
+//
+// Used in combination with MutabilityMode.ALLOW_FORCED to enable forced removal.
+message DeleteByIDWithForce {
```

Comment:
I was thinking that maybe this should be generalized to `DeleteByIDWithOptions`, as eventually we might want to add new delete options and then naming becomes a mess. However, that then leads to the question what we do if a delete handler doesn't understand any of the options (right now, with force being the only option, we make it very explicit in the service signature what options we accept). Probably fine to keep for now, we can always change the name if we start adding more options.